src/AWS.Deploy.CLI/Commands/DeleteDeploymentCommand.cs

```diff
@@ -66,6 +69,7 @@ public async Task ExecuteAsync(string stackName)
                 var _ = monitor.StartAsync();
 
                 await WaitForStackDelete(stackName);
+
```

Comment:
Should we be removing the stack from `aws-deployments.json` after we delete it?

Reply:
Originally I was doing that but since modifying the aws-deployments.json file needs knowledge of the project path and the delete command does not have it, I decided against it. The alternative would be to modify the delete command to only run in a directory with a valid project (which i don't think we want to do)

Reply:
Seems like we could do a best attempt. If the current directory or if `--project-path` is set then we can check the local deployment file. If the stack is not listed or the file is not found then we don't do any clean up. Currently the delete command does still take in `--project-path`.

Reply:
Done